plugin/storage/cassandra/options.go

```diff
@@ -57,6 +58,7 @@ type Options struct {
 	primary                *namespaceConfig
 	others                 map[string]*namespaceConfig
 	SpanStoreWriteCacheTTL time.Duration
+	writesPerSecond        int
```

Comment:
Shouldn't this be public? And qualified with `SpanStore` prefix.

Reply:
IDK why SpanStoreWriteCacheTTL is public either. There is no serialization here. I think it might be a workaround to pass options to another package that need to read `SpanStoreWriteCacheTTL`, but doesn't make much sense given that passing the `int` is just as effective.

Reply:
how are we using this internally? SpanStoreWriteCacheTTL is externally settable option.

Reply:
Both of these are initialized by command-line flags, so neither should be public. I did a code search and could not find a reference to SpanStoreWriteCacheTTL (not even in our internal Uber repo). Seems to be vestigial from some point where we needed access from our internal repo into this part of the OSS.

Reply:
We don't use cli flags internally. If we need to set these two fields how would we do it without them being public? Did you try your branch without our internal main?

Reply:
```
$ git grep -l SpanStoreWriteCacheTTL
plugin/storage/cassandra/factory.go
plugin/storage/cassandra/options.go
```